### PR TITLE
Add Rockhound

### PR DESCRIPTION
**Pull request by santisoler**
Checklist

- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages
- [ ] Build number is 0
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/rockhound/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      xarray). Many of these models use non-conventional file formats or can be tricky
+      to find on the internet. RockHound knows how to download them if you don't
+      already have them locally, read the file format, and return a nicely formatted
+      data structure. Under the hood, it uses Pooch to manage the downloads.
+  doc_url: https://www.fatiando.org/rockhound
+  dev_url: https://github.com/fatiando/rockhound
+
+extra:
+  recipe-maintainers:
+    - leouieda
```
Comment (chrisburr) [inline recipes/rockhound/meta.yaml]: @leouieda Are you happy to be listed as a maintainer of this recipe?

File: recipes/rockhound/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+  run:
+    - python >=3.5
+    - pooch>=0.5
```
Comment (chrisburr) [inline recipes/rockhound/meta.yaml]: There needs to be a space here:
```suggestion
    - pooch >=0.5
```

File: recipes/rockhound/meta.yaml (hunk 1 above)
Comment (leouieda) [reply recipes/rockhound/meta.yaml]: @chrisburr yep 👍 Sorry, I should have left a comment here about this from the start.

**Comment by marcelotrevisani** [comment]
LGTM! Thanks for added this to conda-forge! :tada:

**Pull request merged**